tests/stdlib/tdecodequery.nim

```diff
@@ -0,0 +1,10 @@
+import uri, sequtils
+
+
+block:
+  const queryString = "a=1&b=0"
+  const wrongQueryString = "a=1&b=2c=6"
+
+  doAssert toSeq(decodeQuery(queryString)) == @[("a", "1"), ("b", "0")]
```

Comment:
you're only using `queryString` and `wrongQueryString`, so it's better to inline them: more readble.

```nim
doAssert toSeq(decodeQuery("a=1&b=0")) == @[("a", "1"), ("b", "0")]
doAssertRaises(UriParseError):
  discard toSeq(decodeQuery("a=1&b=2c=6"))
```

Reply:
same as above